deploy-manage/toc.yml

```diff
@@ -456,16 +456,21 @@ toc:
           - file: security/secure-your-eck-installation.md
       - file: security/secure-your-cluster-deployment.md
         children:
+          - file: security/security-certificates-keys.md
+          - file: security/set-up-minimal-security.md
```

Comment:
consider grouping these under a self-managed category or prefacing them with self-managed in the sidebar. they currently look like they apply to all deployment types from the sidebar (especially because they're at the top)